pkg/webhook/config.go

```diff
@@ -318,16 +320,28 @@ func parseVaultConfig(obj metav1.Object, ar *model.AdmissionReview) VaultConfig
 		vaultConfig.AgentOnce = false
 	}
 
-	if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-cpu"]); err == nil {
-		vaultConfig.AgentCPU = val
+	if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-cpu-limit"]); err == nil {
+		vaultConfig.AgentCPULimit = val
 	} else {
-		vaultConfig.AgentCPU = resource.MustParse("100m")
+		vaultConfig.AgentCPULimit = resource.MustParse("100m")
 	}
 
-	if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-memory"]); err == nil {
-		vaultConfig.AgentMemory = val
+	if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-memory-limit"]); err == nil {
+		vaultConfig.AgentMemoryLimit = val
 	} else {
-		vaultConfig.AgentMemory = resource.MustParse("128Mi")
+		vaultConfig.AgentMemoryLimit = resource.MustParse("128Mi")
+	}
```

Comment:
This is going to break compatibility, can we use the `vault.security.banzaicloud.io/vault-agent-memory` as main and `vault.security.banzaicloud.io/vault-agent-memory-limit` as a backup option, e.g.
```go
if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-memory"]); err == nil {
	vaultConfig.AgentMemoryLimit = val
} else if val, err := resource.ParseQuantity(annotations["vault.security.banzaicloud.io/vault-agent-memory-limit"]); err == nil {
	vaultConfig.AgentMemoryLimit = val
} else {
	vaultConfig.AgentMemoryLimit = resource.MustParse("128Mi")
}
```